pyccel/codegen/printing/ccode.py

```diff
@@ -787,7 +787,22 @@ def _print_NativeComplex(self, expr):
     def _print_NativeString(self, expr):
         return 'string'
 
-    def function_signature(self, expr):
+    def function_signature(self, expr, print_arg_names = True):
```

Comment:
I think the default should be False, as in most places we don't need the argument names (.h file, function as argument arguments)

Reply:
i was thinking the special case here is not printing arg name 
given that we call function signature in a lot of different places

Reply:
Ah ok. I thought it was only called from `_visit_FunctionDef` (True), `_visit_ModuleHeader` (currently True, but it wouldn't be unusual if it was False here, see @yguclu 's assumption above) and itself (False)

Reply:
Actually the [consensus](https://stackoverflow.com/questions/7891526/should-function-declarations-include-parameter-names) seems to be that it is better for documentation/readability purposes to include the argument name in the .h file so I agree that the default should be `True`